distribution/asf-release-process-guide.md

```diff
@@ -623,36 +623,41 @@ http://www.apache.org/legal/release-policy.html#release-announcements
 
-```yaml
-druid_versions:
-  - release: 0.16
-    versions:
-      - version: 0.16.0-incubating
-        date: 2019-09-24
-  - release: 0.15
-    versions:
-      - version: 0.15.1-incubating
-        date: 2019-08-15
-```
+For more thorough instructions and a description of what the `do_all_things` script does, see the [`druid-website-src` README](https://github.com/apache/druid-website-src)
 
-3. From druid-website, checkout branch `asf-site`.
+1. Make sure you pull the latest commits from the `druid` release branch. There have likely been backported documentation PRs between the initial staging date and the release date.
 
-4. From druid-website-src, checkout the branch you created to update the staged Druid website.
+2. Pull the latest from `https://github.com/apache/druid-website` and `https://github.com/apache/druid-website-src`. These repositories should be in the same directory as your Druid repository that should have the release tag checked out.
 
-5. From druid-website-src, run `./release.sh 0.17.0 0.17.0`, replacing `0.17.0` where the first argument is the release version and 2nd argument is commit-ish. This script will:
+3. From `druid-website`, create a release branch based on the latest `asf-site` branch.
 
-* checkout the tag of the Druid release version
-* build the docs for that version into druid-website-src
-* build druid-website-src into druid-website
-* stage druid-website-src and druid-website repositories to git.
+4. From `druid-website-src`, check out a new branch for the release:
+   
+   1. Update the version list in `static/js/version.js` with the version you're releasing and the release date. The highest release version goes in position 0. Make sure to remove older releases. We only keep the 3 most recent listed in the file.
+   1. In `scripts`, run: 
+   
+   ```python
+   # This copies the Markdown files from `druid` and builds the website using them
+   # Include `--skip-install` if you already have Docusaurus 2 installed in druid-website-src. 
+   # The script assumes you use `npm`. If you use `yarn`, include `--yarn`.
 
-6. Make a PR to the src repo (https://github.com/apache/druid-website-src) for the master branch.
+   python do_all_things.py -v VERSION --source /my/path/to/apache/druid
+   ```
    
-7. Make a PR to the website repo (https://github.com/apache/druid-website) for the `asf-site` branch. Once the website PR is merged, https://druid.apache.org/ will be updated immediately.
+
+5. Add the files to a PR to the src repo (https://github.com/apache/druid-website-src) for the release branch you just created. In the changed files, you should see the following:
+  - In `published_versions` directory: HTML files for `docs/VERSION` , `docs/latest`, and assorted HTML and non-HTML files. 
+  - In the `docs` directory at the root of the repo, the new Markdown files.
+    
+    All these files should be part of your PR to `druid-website-src`.
+   <br />
```

Comment:
```suggestion
    All these files should be part of your PR to `druid-website-src`.  
```
Markdown isn't being rendered properly in the "Verify the site..." line

![image](https://github.com/317brian/druid/assets/7747997/888a347d-2b8b-4e9b-87b2-435392941e1a)